go/cmd/vtctldclient/command/transactions.go

```diff
@@ -120,6 +141,22 @@ func commandConcludeTransaction(cmd *cobra.Command, args []string) (err error) {
 	return err
 }
 
+func commandReadTransactionState(cmd *cobra.Command, args []string) error {
+	cli.FinishedParsing(cmd)
+
+	rts, err := client.ReadTransactionState(commandCtx,
+		&vtctldatapb.ReadTransactionStateRequest{
+			Dtid: readTransactionStateOptions.Dtid,
+		})
+
+	if err != nil || rts == nil {
+		return err
+	}
+
+	fmt.Println(string(rts.String()))
```

Comment:
Other commands are emitting JSON output, is `rts` in json?
Also the error is in json output format.

Reply:
rts output is json yes. The error is not in json output for other commands too.

Reply:
It is for `conclude`
```
func commandConcludeTransaction(cmd *cobra.Command, args []string) (err error) {
...
	_, err = client.ConcludeTransaction(commandCtx,
		&vtctldatapb.ConcludeTransactionRequest{
			Dtid: concludeTransactionOptions.Dtid,
		})
	if err != nil {
		output.Message = concludeFailure
		output.Error = err.Error()
	}

```

Reply:
Its not for `commandGetUnresolvedTransactions`. I don't see why the Error has to be in json format.

Reply:
Fixed